### ISSUE TITLE
Enable social logins like google/facebook/apple for the login page.

### ISSUE THREAD

**Issue by pradeepmantha** [expo-community/expo-firebase-starter#52]
Hi, 
Its not an issue, i would like to add this as a feature request.

Thanks for the template. Would it be great to enable Social logins like google/facebook/apple etc? as its a common requirement for almost all apps.

Thanks

**Comment by pradeepmantha** [comment]
Something like doordash app, would be great.

**Comment by amandeepmittal** [comment]
Great idea @pradeepmantha! Thanks for sharing. Will look into supporting it soon.

**Comment by pradeepmantha** [comment]
Please make sure you use latest expo-auth-session & firebase api for authentication

**Comment by flexbox** [comment]
Hey @pradeepmantha 

we have a new release https://github.com/expo-community/expo-firebase-starter/releases/tag/0.16.0
If your problem is solved can you please close the issue? 
It sparks joy for maintainers ✨

**Comment by pradeepmantha** [comment]
Nope, i don't see the new changes, or the documentation updated that shows how to enable them. So this feature request still need to be worked on.

**Comment by amandeepmittal** [comment]
@pradeepmantha, there will be some changes in this starter kit and I plan to address the issue you raised with that. Meanwhile, we can have this PR opened for reference. Thank you for opening it though!

**Comment by jasonhjohnson** [comment]
Hi @amandeepmittal! Thanks for putting this template together it's awesome! I too was wondering if there's been any traction on adding social logins?

Thanks,
Jason
